package.json

```diff
@@ -22,5 +22,10 @@
         "sass-loader": "^7.1.0",
         "vue": "^2.5.17",
         "vue-template-compiler": "^2.5.21"
+    },
+    "dependencies": {
+        "uiv": "^0.28.0",
+        "vue-date-picker": "^2.0.1",
+        "vue2-datepicker": "^2.7.0"
```

Comment:
Please remove all dependencies you don't use. Implement datapicker and keep only what you use.